edition-archive-frontend/components/TabsCard.vue

```diff
@@ -6,7 +6,14 @@
           <a class="nav-link"
              :class="tab === model.currentTab ? 'active' : ''"
              v-on:click="changeToTab(tab)"
-             href="#">{{ tab.title }}</a>
```

Comment:
Hier ist die falsche Stelle dafür.

Reply:
Ja, vermutlich.
Was wäre besser?

tab.title enthält zwar die richtige Übersetzung, aber nur beim Laden der Seite und reagiert nicht auf den Sprachumschalter. Mit der Verwendung der id als Fallunterscheidung fühlte ich mich auf der richtigen Seiten.

Reply:
Die TabsCard.vue soll ja eine Komponente sein die man nachnutzen kann. Jetzt sind da die Titel hart verdrahtet.
Es scheint so als ob das Ergebniss von i18n.t nicht "reactive" ist. Kannst du mal folgendes einfügen in die [[searchType]].vue nach dem tabData Teil einzufügen:

```
watch(i18n.locale, () => {
 for(const td of tabData.value){
   switch (td.id) {
      case BASIC_SEARCH_TYPE:
        td.title = i18n.t('search.basic');
        break;
      case EXTENDED_SEARCH_TYPE:
        td.title = i18n.t('search.extended');
        break;
    }
  }
});
```